addon/services/alexandria-documents.js

```diff
@@ -212,19 +214,21 @@ export default class AlexandriaDocumentsService extends Service {
           return true;
         } catch (error) {
           document.category = previousCategory;
-
-          new ErrorHandler(this, error).notify();
-
-          return false;
+          if (error.errors?.[0]?.status !== "403") {
+            new ErrorHandler(this, error).notify();
+            return false;
```

Comment:
Hey, dropping in from the gitlab mr here. Why do we return false here but if it's a 403 we return a object? Would be nice if the return value of this function would be a bit more fixed and predictable. IMO the function signature can be broken down to something like this `(category, docIds) => undefined | { error: str | int, document }` What do you think?

Reply:
yes, the error handling and return values are quite inconsistent, even before, it was a string or boolean and also the error notification shouldn't be in here, i guess this should be a bit bigger refactoring than just changing the return type structure of both methods, to tackle those problems we opened issue #1192